driver/src/main/scala/api/commands/instanceadministration.scala

```diff
@@ -35,10 +35,10 @@ case class CollStats(scale: Option[Int] = None) extends CollectionCommand with C
  * @param size The size in bytes (or in bytes / scale, if any).
  * @param averageObjectSize The average object size in bytes (or in bytes / scale, if any).
  * @param storageSize Preallocated space for the collection.
- * @param numExtents Number of extents (contiguously allocated chunks of datafile space).
+ * @param numExtents Number of extents (contiguously allocated chunks of datafile space, only for mmapv1 storage engine).
```

Comment:
Some stat properties are now optional (only present if using the mmapv1 storage engine)